deps/v8/include/v8-internal.h

```diff
@@ -209,8 +230,12 @@ class Internals {
       kIsolateFastCCallCallerFpOffset + kApiSystemPointerSize;
   static const int kIsolateFastApiCallTargetOffset =
       kIsolateFastCCallCallerPcOffset + kApiSystemPointerSize;
-  static const int kIsolateStackGuardOffset =
+  static const int kIsolateCageBaseOffset =
       kIsolateFastApiCallTargetOffset + kApiSystemPointerSize;
+  static const int kIsolateLongTaskStatsCounterOffset =
+      kIsolateCageBaseOffset + kApiSystemPointerSize;
+  static const int kIsolateStackGuardOffset =
+      kIsolateLongTaskStatsCounterOffset + kApiSizetSize;
   static const int kIsolateRootsOffset =
       kIsolateStackGuardOffset + 7 * kApiSystemPointerSize;
```

Comment:
This changes the value of `kIsolateRootsOffset` – that’s an ABI-breaking change, unfortunately

Reply:
This comes from https://github.com/v8/v8/commit/521ae93bf652b0cc1fec1fdd788e3b80fb72a349
I guess we can just revert it?

Reply:
It's reverted

Reply:
Should be fine, yes :+1: